README.md

```diff
@@ -2,136 +2,130 @@
 
 ## About
 
-This project allow to allocate mac addresses from a pool to secondary interfaces using
+This project allows to allocate mac addresses from a pool to secondary interfaces using
 [Network Plumbing Working Group de-facto standard](https://github.com/k8snetworkplumbingwg/multi-net-spec).
 For VirtualMachines, it also allocates a mac address for the primary interface if [masquerade](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=masquerade) or [slirp](https://kubevirt.io/user-guide/#/creation/interfaces-and-networks?id=slirp) binding mechanism is used.
 
-## Usage
+## The Why
 
-For test environment you can use the [development environment](#Develop).
+Kubemacpool allows for collision-free Mac allocation to Pods and VirtualMachines, in a a Kubernetes cluster operating 
```

Comment:
Let's be consistent with mac. Either use "mac" or "MAC" everywhere

Reply:
Done